warehouse/query-core/src/test/java/datawave/query/jexl/visitors/IngestTypePruningVisitorTest.java

```diff
@@ -438,7 +441,7 @@ public void testEvaluationOnlyField() {
     @Test
     public void testPruneNegation() {
         String query = "A == '1' || !((_Delayed_ = true) && (A == '1' && C == '2'))";
-        test(query, "A == '1'");
+        test(query, query);
```

Comment:
Does this indicate we are not pruning the negation, if the expected now has the negation included?

Reply:
Previously the negation would self-prune, let me take a closer look at why this stopped visiting the negated branch..